src/diffusers/models/attention_processor.py

```diff
@@ -2697,3 +2809,91 @@ def __call__(
     LoRAXFormersAttnProcessor,
     LoRAAttnAddedKVProcessor,
 ]
+
+from typing import Tuple
```

Comment:
put it here for now (I don't think they belong here)

Reply:
Hi I tested this branch and it gives the same outputs as the old pipeline with the same generator. The code looks good to me.